### PR TITLE
[WIP] UI Inventory

### PR DESCRIPTION
**Pull request by pinkiebell**
#1738 

- [x] Alerts
- [x] Animations/Loaders
- [x] Buttons
- [x] Forms
- [x] Typography (H1-H6, body copy, etc)
- [ ] Search Field
- [ ] Color Swatches
- [x] Iconography
- [ ] Collapse/Expanders
- [x] Images (in various sizes)
- [x] Cards
- [x] Carousels
- [x] Checkboxes
- [x] Button Groups
- [ ] Dropdown Menus
- [ ] Data Tables
- [x] Illustrations (show all)
- [x] Page Headers (show an example of each)
- [x] Dialogs

### CODE REVIEW

**Comment by owocki** [comment]
let me know when this isnt WIP anymore and ill review

**Comment by pinkiebell** [comment]
@owocki Ready for review.
If I forgot a UI element, let me know.

File: app/app/urls.py [hunk 1]
```diff
@@ -178,6 +178,7 @@
     # brochureware views
     url(r'^about/?', retail.views.about, name='about'),
     url(r'^mission/?', retail.views.mission, name='mission'),
+    url(r'^ui/?', retail.views.ui, name='ui'),
```
Comment (SaptakS) [inline app/app/urls.py]: Please use `re_path` since we are trying to migrate completely from `url`

File: app/retail/templates/ui_inventory.html [hunk 2]
```diff
@@ -0,0 +1,748 @@
...
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not,see
+<http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
```
Comment (mbeacom) [inline app/retail/templates/ui_inventory.html]: Can you wrap relevant copy with `{% trans "" %}` ?

File: app/app/urls.py (hunk 1 above)
Comment (mbeacom) [reply app/app/urls.py]: ^

**Comment by thelostone-mc** [comment]
@PixelantDesign @mbeacom @SaptakS 

This is the how the page looks at the moment

screenshot_2018-07-18 ui inventory gitcoin

**Comment by thelostone-mc** [comment]
@pinkiebell lol thats cause I took a full page screenshot and the animation kicks in only when you actually scroll  :P 

Btw just synced up with @PixelantDesign 

- Could you tick whatever you've covered on the checklist (mentioned in the issue)
- Also could you throw in the code snippet below each component so that folks know how to use it 
  (don't worry about how it looks, we'll fix it up in iterative-ly  )

**Comment by thelostone-mc** [comment]
@pinkiebell wohoo ^_^

x1

@PixelantDesign Merge ready ? 
@pinkiebell has checked out whatever she's added in the list

@SaptakS / @mbeacom If in case you guys merge it before I get to it -> we need to change to uri to `/alpha-ui-styleguide` so that it's hard to find :P (As advised by @PixelantDesign )

**Comment by SaptakS** [comment]
I am just curious if giving an icons like [here](https://material-ui.com/demos/text-fields/) to show the code is more intuitive than clicking on the card.

**Comment by pinkiebell** [comment]
Just a quick rebase and I added some more assets + style fixes.
@SaptakS 
Isn't it a bit too much If we show a icon on every example?

**Comment by PixelantDesign** [comment]
Looks good thank you @pinkiebell

File: app/retail/templates/ui_inventory.html [hunk 3]
```diff
@@ -0,0 +1,921 @@
...
+          </div>
+          <div class="col-12">
+            <h5 class="text-center font-title">{% trans "Modal Dialog" %}</h5>
+          </div>
+        </div>
+      </div>
+
+      <div class="container-fluid p-5 assets">
+        <p class="h1">{% trans "Vector Graphics" %}</p>
+        <img src="{% static "v2/images/quickstart-box.svg" %}"/>
```
Comment (mbeacom) [inline app/retail/templates/ui_inventory.html]: I am not saying we need to do it here... but we'll probably want to modify the vast majority of the asset handling and repetitive entries here to simply loop over assets  in `images/` and the associated subs / context data.  We'd drop the line count in this file down by quite a bit and we wouldn't have to manually add every new asset as they're added (I'd be willing to bet it won't be long until we're missing assets here).

Comment (pinkiebell) [reply app/retail/templates/ui_inventory.html]: That would be very nice. I'm not familiar with Django, so I will dig into that today. 👍

**Comment by thelostone-mc** [comment]
@SaptakS meh it's just for a first pass -> we'll improve it in cycles 
I guess the scope of this task was just to get all the components together on one page

I already bugged @pinkiebell to add the code snippets (even though I believe it wasn't part of the original task :P )

**Pull request merged**